clients/tabby-agent/src/chat/inlineEdit.ts

```diff
@@ -189,13 +207,16 @@ export class ChatEditProvider implements Feature {
                 return userCommand;
               case "{{languageId}}":
                 return document.languageId;
+              case "{{fileContext}}":
```

Comment:
@wsxiaoys we should update prompt template to include `{{fileContext}}` part. Can you guide me on that?